pkg/gateway/gateway.go

```diff
@@ -366,7 +387,23 @@ func (g *gatewayModule) ensureGateway(ctx context.Context, forceResstart bool) (
 	if err != nil {
 		return pkg.PublicConfig{}, errors.Wrap(err, "failed to check traefik status")
 	}
-	if running && forceResstart {
+	path, name, err := g.traefikBinary(ctx)
```

Comment:
What if this is the first boot and the service was never configured. this will Always fail because u don't check if the file does not exist